typescript/commands/quick_info.py

```diff
@@ -47,20 +50,24 @@ def handle_quick_info(self, quick_info_resp_dict):
                     sublime.active_window().run_command('show_panel', {'panel': 'output.doc'})
                 status_info_str = info_str + " (^T^Q for more)"
             self.view.set_status("typescript_info", status_info_str)
+
+            # process display string
             if TOOLTIP_SUPPORT:
                 html_info_str = escape_html(info_str)
                 html_doc_str = escape_html(doc_str)
                 html = "<div>" + html_info_str + "</div>"
                 if len(doc_str) > 0:
                     html += "<div>" + html_doc_str + "</div>"
-                self.view.show_popup(html, location=-1, max_width=800)
+                self.view.show_popup(html, flags=sublime.HIDE_ON_MOUSE_MOVE_AWAY, location=self.display_point, max_width=800)
```

Comment:
Does `HIDE_ON_MOUSE_MOVE_AWAY` have any impact if the popup was manually triggered? (i.e. not from hover)?